x-pack/plugin/src/test/resources/rest-api-spec/test/runtime_fields/30_double.yml

```diff
@@ -0,0 +1,203 @@
+---
+setup:
```

Comment:
do we have tests somewhere to verify that the scripted fields work fine when used for sorting? Maybe we should also have tests for when they get referred to as part of scripts? I would not cover these two scenarios in a yaml test though.

Reply:
We do have a scenario where a scripted field refers to a another one, but not where a script query refers to a scripted field. I'll add that.